public/class-wp-agora-io-public.php

```diff
@@ -375,6 +375,8 @@ public function enqueue_scripts() {
 
 		wp_enqueue_script( $this->plugin_name, plugin_dir_url( __FILE__ ) . 'js/wp-agora-io-public.js', array( 'jquery' ), $this->version, false );
 
+		wp_enqueue_script( $this->plugin_name.'-agora-deviceTest-js', plugin_dir_url( __FILE__ ) . 'js/wp-agora-io-device-test.js', array( 'jquery' ), $this->version, false );
```

Comment:
are we loading this script every time? this should only get enqueued if the pre-call is enabled.

Reply:
> are we loading this script every time? this should only get enqueued if the pre-call is enabled.

Thanks. Will do that in next PR.

Reply:
Done. Kindly check it.